apps/website/blog/2024-11-20-maci-platform.md

```diff
@@ -0,0 +1,93 @@
+
+Projects applying to be part of a round are able to submit the required information via a form, which will be customizable by the round operator to include all required and relevant information for that specific round. Approvals of those projects will be at the discretion of the operators.
+
+Finally, users/voters will be able to access the voting rounds only if they pass the gatekeeping restrictions that are in place. Currently, it is possible to gate voting rounds using different strategies:
+
+- allowing anyone to participate
+- accepting only users with a certain [EAS](https://docs.attest.sh/docs/) attestation
+- accepting only users part of a [Semaphore](https://semaphore.pse.dev/) group
+- accepting only users with a certain [Hat](https://docs.hatsprotocol.xyz/)
+- accepting only users with a certain [Zupass](https://github.com/proofcarryingdata/zupass) credential
+
+...and many more to come. Please refer to [MACI’s gatekeeper page](https://maci.pse.dev/docs/technical-references/smart-contracts/Gatekeepers)
+
+### Roadmap
+
+What have we built so far? As of now, we have just released a [version 1](https://github.com/privacy-scaling-explorations/maci-platform/releases/tag/v1). This release focuses on voter and applicant project experience, enabling them to interact smoothly with the platform by signing up, submitting applications, and voting. Finally, users can also see results once the votes have been tallied by the coordinator.
+
+V2 on the other hand, will focus on the round organizer experience. At a high level, the core devs are focusing on implementing the following features:
+
+- Dashboard for contract deployment and round configuration
+- Allowing multiple rounds to live within one MACI Platform instance
+- Payout disbursement via the interface by either the voters, applicants, or operators
+- Full integration with [MACI’s coordinator service](https://github.com/privacy-scaling-explorations/maci-platform/issues/278) to allow finalization of rounds within the interface
+- Continued integration with different gatekeepers like [Gitcoin Passport](https://docs.passport.xyz/)
+
+Furthermore, we plan to experiment with AA (account abstraction) to enable smooth experiences for both users and organisers. For users, we have implemented an end-to-end flow using an embedded wallet and paymaster as part of PSE’s internal [hackathon](https://github.com/privacy-scaling-explorations/hacking-pse/tree/main/packages/interface). For organizers we will be using a smart account and [session keys](https://docs.zerodev.app/sdk/advanced/session-keys) to safely interact with the coordinator service and interact with MACI contracts.
```

Comment:
should we also mention the paymaster we used in the devcon round here?

Reply:
yea will add